shared/wallets/receive-modal/index.js

```diff
@@ -1,15 +1,6 @@
 // @flow
 import * as React from 'react'
-import {
-  Box2,
-  Button,
-  CopyText,
-  Icon,
-  InfoNote,
-  MaybePopup,
-  Text,
-  iconCastPlatformStyles,
-} from '../../common-adapters'
+import * as Kb from '../../common-adapters'
```

Comment:
Why this change? I haven't seen this pattern anywhere else...

Reply:
It's the new hotness.

Reply:
But it makes the markup distinctly less readable...

Reply:
I think this ultimately boils down to a stylistic choice. I don't find namespaces components really that unreadable. We have started using this style recently. I find that it's just easier development wise instead of having to keep managing the imports and add all this vertical space to pull in the same components over and over

Reply:
I like it. I assumed we did it the old way because there was some advantage to not importing everything like that.